### PR TITLE
Adding VCub 

### PR DESCRIPTION
**Pull request by trouyer**
Hello,

I  have opened this pull request  to add VCub the bike share system of Bordeaux Métropole.

Best wishes

Thomas

### CODE REVIEW

File: pybikes/keolis.py [hunk 1]
```diff
@@ -204,3 +205,50 @@ def __init__(self, fields):
...
+        self.name = fields['name']
+        self.latitude = float(fields['latitude'])
+        self.longitude = float(fields['longitude'])
+        e_bikes = int(fields['nbElectricBikeAvailable'])
+        manual_bikes = int(fields['nbBikeAvailable'])
+        self.bikes = self.e_bikes + self.manual_bikes
+        self.free = int(fields['nbPlaceAvailable'])
+        self.extra = {
+            'status': int(fields['status']), # 0: maintenance, 1: operating
+            'slots': self.bikes + self.free,
```
[the remark this answers is not in the document]
Comment (trouyer) [reply pybikes/keolis.py]: You're right. I think we can skip it also when station is offline.

File: pybikes/data/keolis.json [hunk 2]
```diff
@@ -48,6 +48,22 @@
...
+            "instances": [
+                {
+                    "tag": "vcub",
+                    "meta": {
+                        "country": "FR",
+                        "city": "Bordeaux",
+                        "latitude": 44.837789,
+                        "longitude": -0.57918,
+                        "company": ["Keolis", "Bordeaux Métropole"],
+                        "name": "VCub"
```
[the remark this answers is not in the document]
Comment (trouyer) [reply pybikes/data/keolis.json]: VCub is the name of Keolis bike system make for Bordeaux. The name is a wordplay with the name of the urban agglomeration of Bordeaux (Communauté Urbaine de Bordeaux) and the power of three. VCub seems to be old  (the Communauté Urbaine de Bordeaux was renamed to Bordeaux Métropole in 2015), V3 is the new one.

File: pybikes/data/keolis.json [hunk 3]
```diff
@@ -48,6 +48,22 @@
                     "dataset": "vls-stations-etat-tr"
                 }
             ]
+        },
+        "VCub": {
+            "instances": [
+                {
+                    "tag": "vcub",
```
[the remark this answers is not in the document]
Comment (trouyer) [reply pybikes/data/keolis.json]: V3 is a specific name for Bordeaux. I don't think Keolis will use this name for another city.

**Comment by trouyer** [comment]
Hi,

thanks for this detailled review. I've pushed changes.

**Comment by eskerda** [comment]
Hey @trouyer , sorry for the delay. Looks good, thanks!

**Pull request merged**